homeassistant/components/launch_library/config_flow.py

```diff
@@ -0,0 +1,23 @@
+"""Config flow to configure launch library component."""
+from homeassistant import config_entries
+
+from .const import DOMAIN
+
+
+class LaunchLibraryFlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
+    """Config flow for Launch Library component."""
+
+    VERSION = 1
+
+    async def async_step_user(self, user_input=None):
+        """Handle a flow initialized by the user."""
+        # Check if already configured
+        await self.async_set_unique_id(DOMAIN)
+        self._abort_if_unique_id_configured()
```

Comment:
No need for a unique ID in this, just abort if any if exist

Reply:
You mean like this? ( I am still new to coding a HA)

````
if self._async_current_entries():
            return self.async_abort(reason="single_instance_allowed")
```